packages/ses/src/transforms.js

```diff
@@ -95,6 +95,7 @@ export function rejectImportExpressions(src) {
 
 const someDirectEvalPattern = new RegExp('\\beval\\s*(?:\\(|/[/*])');
 
+// Exported for unit tests.
```

Comment:
Thanks for noting that.

For all these, what is needed so that the error here could carry more context that just the line number? (not suggesting any such change for this PR though.)

Reply:
We could change the non-capture group for the body of the eval expression in the regular expression to a capture-group and report that. I imagine that could get quite unwieldy depending on the length of the expression.